src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -279,7 +291,11 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
 
             if (result.getParseErrorMessage() == null) {
-                processTree(result.getTree());
+                final DetailNode javadocTree = result.getTree();
+                if (doesCheckAcceptJavadocWithUnclosedHtmlTag()
+                        || !JavadocUtils.hasUnclosedTag(javadocTree)) {
```

Comment:
@romani 
> NonTightHtml

What is non-tight html? this code here is referring to unclosed tags and has other methods already named around that. IE: `AbstractJavadocCheck.java.doesCheckAcceptJavadocWithUnclosedHtmlTag`.